DonjinKrawler-Client/src/main/java/donjinkrawler/Client.java

```diff
@@ -43,58 +45,17 @@ private void setupKryo() throws IOException {
         kryoClient.start();
         kryoClient.connect(5000, serverAddress, SERVER_TCP_PORT, SERVER_UDP_PORT);
 
-        kryoClient.addListener(new Listener() {
+        PacketControlUnit pcu = new PacketControlUnit();
```

Comment:
"Use Intention-Revealing Names". 
The name doesn't really tell me what it is or what pattern it utilises. Instead of Unit, maybe PacketControlChain would suffice better? Not sure if this is the best name to pick, so give some thought.

Reply:
PacketControlChain it is then.